build/ris-fetch.js

```diff
@@ -0,0 +1,171 @@
+'use strict'
+
+import fs from 'node:fs';
+import {stringify} from 'qs'
+import {TMP_PATH, STADA_FILE} from './consts.js';
+
+const BBOX = { // should be approx. quadratic
+    north: 70.0,
+    west: -10.0,
+    south: 35.0,
+    east: 40.0,
+}
+
+const LIMIT_RESULTS = 10000; // maximum number of results allowed by RIS::Stations
+const PADDING_RATIO = 1.1;
+const MAX_DEPTH = 8;
+const MAX_REQUESTS = 1000;
+
+function sleep(ms) {
+    return new Promise((resolve) => {
+        setTimeout(resolve, ms);
+    });
+}
+
+const exists = async (f) => {
+    try {
+        await fs.promises.stat(f);
+        return true;
+    } catch (_) {
+        return false;
+    }
+}
+
+const requestJson = async (url, options) => {
+    try {
+        const response = await fetch(url, options);
+        if (!response.ok) {
+            throw new Error(`Failed to fetch from API: ${response.status}`);
```

Comment:
[nitpick]

Add more request details to the error object.

```suggestion
            const err = new Error(`Failed to fetch from API: ${response.status}`);
            err.url = url;
            err.fetchOptions = options;
            err.response = response;
            throw err;
```